images/disks-images-provider/entrypoint.sh

```diff
@@ -34,7 +34,8 @@ num=$(losetup -l | wc -l)
 [ ${num} -gt 100 ] && echo "attached loopdevices have reach limit number(100)" && exit 1
 
 # Put LOOP_DEVICE in /etc/bashrc in order to detach this loop device when the pod stopped.
-LOOP_DEVICE=$(losetup --find --show /local-storage/cirros.img.raw)
+LOOP_DEVICE=$(chroot /host losetup --find --show /mnt/local-storage/cirros.img.raw)
+echo $LOOP_DEVICE
```

Comment:
Leftover debug print?

Reply:
Yes and no, I can remove it, but I think it would be useful for people to see what loopback device was created.